[COLOUR-FOR-POINT]

SAleksiev opened the pull request:
- Update DYDataPoint model with individual color for each point
- Update foregroundColour
- Update comments

SAleksiev committed:
[COLOUR-FOR-POINT]
- Update DYDataPoint model with individual color
- Update foregroundColour
- Update comments

Commit:
Re-implemented and added color / gradient per point to line and bar chart

- removed color property from DYDataPoint
- added several closures to DYLineChartView for color, background color, diameter and stroke style per point
- added gradientPerBar closure to DYBarChartView

DominikButz commented:
Hi Stefan, I have added a commit to the colour-for-point branch. It implements the requested feature but for consistency, I have included the bar chart as well, so now it is possible to set a linear gradient for each bar individually. Moreover, in the line chart it is now also possible to change the point background colour, the point diameter and even the stroke style. 
I used closures for these changes as you originally suggested. 

I will think about how to implement individual colours for the line sections in the next step.

Commit:
Added colorPerLineSegment closure to DYLineChartView initialiser

- Changed Path drawing logic in DYLineChartView to enable individual colors per line segment
- Updated examples
- Updated readme

DominikButz commented:
The latest commit implements individual colours per line section. I hope it works out for your project.

Commit:
Updated change log for version 0.7
